templates/stripes/stripe-attributes.php

```diff
@@ -7,18 +7,29 @@
 <?php if (!$isEmpty) : ?>
   <div class="<?= $this->e($type, 'wrapperClasses') ?>">
     <div class="vssl-stripe-column">
-      <div class="vssl-stripe--attributes--fieldwrap">
-        <?php foreach ($values as $key => $value) : ?>
+      <?php foreach ($values as $key => $value) : ?>
+        <div class="vssl-stripe--attributes-fieldwrap">
           <?php if (!empty($value['html'])) : ?>
             <div class="vssl-stripe--attributes--field vssl-stripe--attributes--field--<?= $key ?>">
-              <div class="vssl-stripe--attributes--label"><?= $key ?>:</div>
+              <?php
+              $attributeDefinition = array_reduce(
+                $attribute_list,
+                function ($found, $def) use ($key) {
+                    return $key === $def['key'] ? $def : $found;
+                },
```

Comment:
I think could be written, which I like better, although I don't care strongly

```suggestion
               fn ($found, $def) => $key === $def['key'] ? $def : $found
```